app/blueprints/template/forms.py

```diff
@@ -3,9 +3,14 @@
 from wtforms.validators import DataRequired, Regexp
 
 
-class TemplateUploadForm(FlaskForm):
-    template_name = StringField("Template Name", validators=[DataRequired()])
-    file = FileField("Upload File", validators=[DataRequired()])
+class TemplateCreateForm(FlaskForm):
+    template_name = StringField("Template name", validators=[DataRequired()])
+    tasklist_name = StringField(
+        "Task name",
+        description="For example, Project management. Applicants will see this in the application",
+        validators=[DataRequired()],
+    )
+    file = FileField("Upload a file", description="Supports JSON files only", validators=[DataRequired()])
```

Comment:
Once https://github.com/communitiesuk/funding-service-design-fund-application-builder/pull/225 is merged, please can you rebase and start using widgets from govuk-frontend-wtf. It would like like this:

```suggestion
class TemplateCreateForm(FlaskForm):
    template_name = StringField("Template name", widget=GovTextInput(), validators=[DataRequired()])
    tasklist_name = StringField(
        "Task name",
        widget=GovTextInput(),
        description="For example, Project management. Applicants will see this in the application",
        validators=[DataRequired()],
    )
    file = FileField("Upload a file", description="Supports JSON files only", GovFileInput(), validators=[DataRequired()])
```

Then we don't need to have our own custom `fileupload` macro (and can start to unwind all of the custom macros in favour of using govuk-frontend-wtf directly).

Reply:
Fixed